### PR TITLE
Made some improvements to the documentation.

### PR DESCRIPTION
**Pull request by TazmanianDI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,103 @@
...
     serviceInstanceID: "YOUR_SERVICE_INSTANCE_ID",
     onLoad: function(instance) {
       instance.render();
     },
-    // The function that this project exports.
+    // The function that this project exports and is contained in servicedesk.bundle.js.
     serviceDeskFactory: window.WebChatServiceDeskFactory,
   };
-  setTimeout(function(){const t=document.createElement('script');t.src='https://web-chat.global.assistant.watson.appdomain.cloud/loadWatsonAssistantChat.js';document.head.appendChild(t);});
+  setTimeout(function(){const t=document.createElement('script');t.src='https://web-chat.global.assistant.dev.watson.appdomain.cloud/versions/' + (window.watsonAssistantChatOptions.clientVersion || 'latest') + '/WatsonAssistantChatEntry.js';document.head.appendChild(t);});
```
Comment (jfmartinez) [inline README.md:100]: This line of sample code points to a dev URI for loading WebChat

Comment (TazmanianDI) [reply README.md:100]: Oooo, very good eye!

File: docs/API.md [hunk 2]
```diff
@@ -1,213 +1,40 @@
...
+   back to web chat. Review the [ServiceDeskCallback](../src/common/types/serviceDeskCallback.ts) for more information.
+
+An implementation template can be found in [../src/example/webChat/client/src/serviceDeskTemplate.ts](../src/example/webChat/client/src/serviceDeskTemplate.ts).
+
+## Interaction flow
+
+Below is a list of the steps and actions that typically occur when a user is connected to a service desk and how web chat interacts with the service desk integration.
+
+1. When web chat is started, it will create a single instance of the service desk integration using the `serviceDeskFactory` configuration property.
+2. A user requests an agent and Watson Assistant returns a "Connect to Agent" response (type=`connecct_to_agent`).
```
Comment (jfmartinez) [inline docs/API.md:21]: `connecct_to_agent` typo

```suggestion
2. A user requests an agent and Watson Assistant returns a "Connect to Agent" response (type=`connect_to_agent`).
```

**Review by jfmartinez — CHANGES_REQUESTED** [review]
Overall looks good, I just have comment on a typo that needs to be resolved

Also, it is good that you removed the docs on the individual ServiceDesk interface functions docs/API.md. Having it documented in the ServiceDesk interface file is more maintainable

**Review by jfmartinez — APPROVED** [review]

**Pull request merged**